style.css

```diff
@@ -468,3 +502,249 @@ body {
     grid-template-columns: repeat(3, 1fr);
   }
 }
+
+@media screen and (min-width: 768px) {
+  header {
+    margin: 1.25rem 2.5rem;
+  }
+
+  p {
+    font-family: 'Poppins', sans-serif;
+    text-align: left;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+  }
+
+  .intro {
+    width: 70%;
+    display: flex;
+    float: left;
+  }
+
+  .intro h1 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    text-align: left;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    width: 100%;
+  }
+
+  .break {
+    display: none;
+  }
+
+  .heading-menu {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    text-align: center;
+    gap: 32px;
+    font-family: 'Poppins', sans-serif;
+    color: #634634;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    margin-right: 155px;
+  }
+
+  .menubars {
+    display: none;
+  }
+
+  .detail {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    padding: 0 80px;
+  }
+
+  .detail p {
+    padding-left: 99px;
+  }
+
+  .connect {
+    float: left;
+    margin-right: 58%;
+    padding: 0 80px;
+  }
+
+  .conect h3 {
+    display: flex;
+    flex-direction: row;
+    align-items: start;
+    justify-content: left;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #ff9b7f;
+  }
+
+  .logos ul {
+    display: flex;
+    gap: 20px;
+    color: #505f79;
+    list-style: none;
+    margin-left: -1rem;
+    float: left;
+  }
+
+  .homepage {
+    background: url(./images/Aboutme-Big-screen.png) no-repeat;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 12px;
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
```

Comment:
- [ ] Although you successfully implemented the necessary changes with excellence :+1: , as previously mentioned, please ensure that the components on the homepage are properly aligned.


https://github.com/AbuTalha3/Portfolio-Setup/assets/105117832/1edc6a67-b2eb-45b9-bfe5-2f71d1c88a69